build.gradle

```diff
@@ -8,10 +8,28 @@ buildscript {
     }
 }
 
+plugins {
+    id "com.github.hierynomus.license" version "0.14.0"
```

Comment:
It would be great if brief instructions about license plugin usage would be included in application `README.md`